common/buildings/wc_common_buildings.txt

```diff
@@ -0,0 +1,691 @@
+	
+	county_modifier = {
+		supply_limit = 500
+		monthly_county_control_change_factor = 0.05
+	}
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.1
+		light_aerial_toughness_mult = 0.1
+		light_aerial_pursuit_muult = 0.1
+		light_aerial_screen_mult = 0.1
+		
+		massive_aerial_damage_mult = 0.06
+		massive_aerial_toughness_mult = 0.06
+		massive_aerial_pursuit_mult = 0.06
+		massive_aerial_screen_mult = 0.06
+		
+		movement_speed = 0.06
+	}
+	
+	province_modifier = {
+		levy_reinforcement_rate = 0.05
+	}
+	
+	next_building = aviary_complex_07
+	
+	ai_value = {
+		base = 5
+		ai_general_building_modifier = yes
+	}
+}
+
+aviary_complex_07 = {
+	construction_time = quick_construction_time
+	
+	can_construct = {
+		building_requirement_castle_city_church = { LEVEL = 04 }
+		scope:holder.culture = {
+			has_innovation = innovation_royal_armory
+		}
+	}
+	
+	can_construct_showing_failures_only = {
+		building_requirement_civilized = yes
+	}
+
+	cost_gold = cheap_building_tier_7_cost
+	
+	levy = 300
+	
+	county_modifier = {
+		supply_limit = 500
+		monthly_county_control_change_factor = 0.05
+	}
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.13
+		light_aerial_toughness_mult = 0.13
+		light_aerial_pursuit_muult = 0.13
+		light_aerial_screen_mult = 0.13
+		
+		massive_aerial_damage_mult = 0.08
+		massive_aerial_toughness_mult = 0.08
+		massive_aerial_pursuit_mult = 0.08
+		massive_aerial_screen_mult = 0.08
+		
+		movement_speed = 0.06
+	}
+	
+	province_modifier = {
+		levy_reinforcement_rate = 0.05
+	}
+	
+	next_building = aviary_complex_08
+	
+	ai_value = {
+		base = 4
+		ai_general_building_modifier = yes
+	}
+}
+
+aviary_complex_08 = {
+	construction_time = quick_construction_time
+
+	can_construct = {
+		building_requirement_castle_city_church = { LEVEL = 04 }
+		scope:holder.culture = {
+			has_innovation = innovation_royal_armory
+		}
+	}
+
+	can_construct_showing_failures_only = {
+		building_requirement_civilized = yes
+	}
+	
+	cost_gold = cheap_building_tier_8_cost
+	
+	levy = 300
+	
+	county_modifier = {
+		supply_limit = 500
+		monthly_county_control_change_factor = 0.05
+	}
+	
+	character_modifier = {
+		light_aerial_damage_mult = 0.15
+		light_aerial_toughness_mult = 0.15
+		light_aerial_pursuit_muult = 0.15
+		light_aerial_screen_mult = 0.15
+		
+		massive_aerial_damage_mult = 0.1
+		massive_aerial_toughness_mult = 0.1
+		massive_aerial_pursuit_mult = 0.1
+		massive_aerial_screen_mult = 0.1
+		
+		movement_speed = 0.07
+	}
+	
+	province_modifier = {
+		levy_reinforcement_rate = 0.05
+	}
+	
+	ai_value = {
+		base = 3
+		ai_general_building_modifier = yes
+	}
+}
+
+### caster college
+
+caster_college_01 = {
+	construction_time = quick_construction_time
+
+	can_construct_potential = {
+		building_requirement_castle_city_church = { LEVEL = 01 }
+	}
+
+	can_construct_showing_failures_only = {
+		building_requirement_civilized = yes
+	}
+	
+	cost_gold = cheap_building_tier_1_cost
+	
+	levy = normal_building_levy_tier_1
```

Comment:
I wouldn't give levy. I doubt colleges can provide them.

If you're looking for replacement, I think income looks better here.